web/src/components/Auth/AuthModal.js

```diff
@@ -1,35 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 import { Modal, Button } from 'semantic-ui-react';
 import { SSO_IDP } from '../../constants/auth';
 import { TEST_IDS } from '../../constants/ui';
 
-// TODO: may remove the prop isAuthenticated
+const StyledModal = styled(Modal)`
+  &&& {
```

Comment:
what does `&&&` mean?

Reply:
hmmm I think I got this mixed up with less & symbol. will double check on if it actually does anything.

Reply:
Found the post I saw long time ago: https://github.com/styled-components/styled-components/issues/501